local/code/datums/quirks/negative_quirks/alexithymia.dm

```diff
@@ -0,0 +1,7 @@
+/datum/quirk/alexithymia
+	name = "Alexithymia"
+	desc = "You cannot accurately assess your feelings."
+	value = -4
+	//mob_trait = TRAIT_MOOD_NOEXAMINE
```

Comment:
No mob trait is given, and this isn't implemented, so this quirk does nothing

Reply:
FUCK